src/dotnet/CarbonAware.Aggregators/src/CarbonAware/ISciScoreAggregator.cs

```diff
@@ -0,0 +1,17 @@
+using CarbonAware.Model;
+using System.Collections;
+
+namespace CarbonAware.Aggregators.SciScore
+
+{
+    public interface ISciScoreAggregator : IAggregator
+    {
+        /// <summary>
+        /// Calculates the average carbon intensity for a given location and time interval.
+        /// </summary>
+        /// <param name="location">Location object representing the location for the carbon intensity data.</param>
+        /// <param name="timeInterval">ISO8601 time interval representing the interval to calculate carbon intensity over.</param>
+        /// <returns>An average value over the specified time interval.</returns>
+        Task<double> CalculateAverageCarbonIntensityAsync(Location location, string timeInterval);
```

Comment:
Noted this is Bill's PR too but we may want to comment expected errors that may be thrown?